src/app/containers/MostRead/Canonical/LastUpdated.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { number, string, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Timestamp from '@bbc/psammead-timestamp-container';
+import { shouldRenderLastUpdated } from '../utilities';
+
+const LastUpdated = ({ locale, prefix, script, service, timestamp }) =>
+  shouldRenderLastUpdated(timestamp) && (
+    <Timestamp
```

Comment:
Was looking at this issue https://github.com/bbc/psammead/issues/3053 and it seems that this problem should not be fixed in Psammead. There is a correct top padding and no bottom padding applied to timestamp in Psammead Most Read component. The issue with padding comes up in Simorgh when you are using `Timestamp` from `@bbc/psammead-timestamp-container` which accepts a `padding` prop that has a default `true` value. Since here you are not overriding the `padding` value, a bottom padding gets applied underneath the timestamp. You could solve this by passing `padding={false}` here. Tried it locally and it works:

<img width="930" alt="Screenshot 2020-02-04 at 14 01 56" src="https://user-images.githubusercontent.com/17802955/73751702-8d812780-4757-11ea-8969-6388021be1ba.png">

Reply:
Great catch, will still need to find a nice way to just stop rendering `TimeStampWrapper` completely when we don't have a timestamp. Again, more fixes for the future.